Day-I-Social-Card/social-card/src/App.js

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import {
```

Comment:
For this practices purposes I liked how you took the initiative to figure out how to install fontawesome as a dependency but in my opinion for this library aka fontAwesome , it is way easier to just add the CDN to the HTML so you wont have to import each font to different icon. The only advantage to adding FontAwesome as a dependency is if you run into any problems with React not wanting to render the icon.